src/playground/Widgets/EmptyState/index.tsx

```diff
@@ -1,55 +1,48 @@
-import { Button } from "@/components/Actions/Button"
+import { WidgetContainer } from "@/experimental/exports"
 import AreaGraph from "@/icons/AreaGraph"
-import { cn } from "@/lib/utils"
+import Cash from "@/icons/Cash"
 import { Button as ShadcnButton } from "@/ui/button"
-import { CardLink } from "@/ui/card"
-import { cva } from "class-variance-authority"
-import { ComponentProps, forwardRef } from "react"
+import { ComponentProps, ExoticComponent, forwardRef } from "react"
 
-export type Variants = "performance" | "salary"
+type Icon = "area-graph" | "cash"
 
 export interface EmptyStateType {
   title: string
   content: string
-  link: string
-  variant: Variants
-  buttons: {
-    label: string
-    variant: ComponentProps<typeof ShadcnButton>["variant"]
-  }[]
+  icon: Icon
+  buttonLabel?: string
+  buttonVariant?: ComponentProps<typeof ShadcnButton>["variant"]
```

Comment:
do we need to have such a granular setup? are there examples where we use different button variants?

Reply:
<img width="441" alt="Screenshot 2024-09-24 at 08 43 27" src="https://github.com/user-attachments/assets/a79a7ca0-309a-4aac-b57e-5460f763c4d8">

Reply:
We could also provide a `promote?: boolean` prop to use the `promote` variant for the button and, if not provided, it will fall into the default variant

Reply:
I like the idea! It will suggest that the whole widget is in the "promotion" state